src/2.6.0/wso2apim.js

```diff
@@ -194,6 +194,7 @@ function constructAPIDef(user, gatewayEnv, apiDef, apiId) {
     }
 
     const wso2ApiDefinition = {
+      id: (apiId) ? apiId: undefined,
```

Comment:
Can `apiId` be something else than `undefined`? I'm thinking `id: apiId` should suffice, because if `apiId`'s not there then it will be undefined. Unless it can be null or other falsy value